kyuubi-server/web-ui/src/views/session/session-detail/index.vue

```diff
@@ -0,0 +1,183 @@
+        <template #default="scope">
+          <el-tag
+            :type="scope.row.state === 'ERROR_STATE' ? 'danger' : 'success'"
+            >{{ scope.row.state }}</el-tag
+          >
+        </template>
+      </el-table-column>
+      <el-table-column :label="$t('create_time')" width="180">
+        <template #default="scope">
+          {{
+            scope.row.createTime != null && scope.row.createTime > -1
+              ? format(scope.row.createTime, 'yyyy-MM-dd HH:mm:ss')
+              : '-'
+          }}
+        </template>
+      </el-table-column>
+      <el-table-column :label="$t('start_time')" width="180">
+        <template #default="scope">
+          {{
+            scope.row.startTime != null && scope.row.startTime > -1
+              ? format(scope.row.startTime, 'yyyy-MM-dd HH:mm:ss')
+              : '-'
+          }}
+        </template>
+      </el-table-column>
+      <el-table-column :label="$t('complete_time')" width="180">
+        <template #default="scope">
+          {{
+            scope.row.completeTime != null && scope.row.completeTime > -1
+              ? format(scope.row.completeTime, 'yyyy-MM-dd HH:mm:ss')
+              : '-'
+          }}
+        </template>
+      </el-table-column>
+      <el-table-column
+        prop="exception"
+        :label="$t('failure_reason')"
+        width="160" />
+    </el-table>
+  </el-card>
+</template>
+<script lang="ts" setup>
+  import { Ref, ref } from 'vue'
+  import { getSession } from '@/api/session'
+  import { getOperations } from '@/api/operation'
+  import { useRoute } from 'vue-router'
+  import { format } from 'date-fns'
+  import { secondTransfer } from '@/utils/unit'
+  import { useTable } from '@/views/common/use-table'
+
+  const route = useRoute()
+  const sessionId = route.query.sessionId
+  const sessionProperties: Ref<any> = ref({})
+  const sessionPropertiesLoading = ref(false)
+  const { tableData, loading, getList: _getList } = useTable()
+
+  function getSessionById() {
+    if (sessionId) {
+      sessionPropertiesLoading.value = true
+      getSession(sessionId as string)
+        .then((res: any) => {
+          sessionProperties.value = res?.conf || {}
+        })
+        .finally(() => {
+          sessionPropertiesLoading.value = false
+        })
+    }
+  }
+  const getList = () => {
+    const sessionId = route.query.sessionId
+    if (sessionId) {
+      _getList(getOperations, { sessionHandle: sessionId })
+    }
+  }
+  getSessionById()
+  getList()
+</script>
+<style lang="scss" scoped>
+  header {
+    display: flex;
+    justify-content: space-between;
+    .el-breadcrumb {
+      line-height: 32px;
+    }
+  }
+  .session-properties-container {
+    margin-bottom: 20px;
+    .collapse-header {
+      margin-left: 18px;
+      display: flex;
+      font-size: 16px;
+      line-height: 24px;
+      font-weight: 400;
+      font-family: 'Myriad Pro', 'Helvetica Neue', Arial, Helvetica, sans-serif;
```

Comment:
why use those fonts?

Reply:
Used to override the style brought by the `.el-collapse-item__header`, consistent with the following `.el-card__header`